lib/nimble_csv.ex

```diff
@@ -278,11 +299,24 @@ defmodule NimbleCSV do
         end
       end
 
+      if @escape_formula != nil do
+        @escape_formula_pattern elem(@escape_formula, 0)
+        @escape_formula_prefix elem(@escape_formula, 1)
+        defp maybe_escape_formulas(data) do
+          case String.starts_with?(data, @escape_formula_pattern) do
+            true -> @escape_formula_prefix
+            false -> []
+          end
+        end
+      else
+        defp maybe_escape_formulas(_data), do: []
+      end
```

Comment:
I am thinking `escape_formula` can be a map or a keyword list, so we can escape different values with different rules. Hence `escape_formula: [{~w(@ + - =), "\t"}]`. Can you please update accordingly? Then this code can be:

```suggestion
      for {keys, value} <- @escape_formula,
          key <- keys do
        defp maybe_escape_formulas(unquote(key) <> _), do: unquote(value)
      end

      defp maybe_escape_formulas(_data), do: []
```

Reply:
I don't think that code works, I looked at the `newlines_separator` macro for inspiration.